apps/pay/cypress/e2e/point-of-sale.cy.ts

```diff
@@ -54,4 +56,46 @@ describe("Point of Sale", () => {
     cy.get("[data-testid=qrcode-container]").as("qrcodeContainer")
     cy.get("@qrcodeContainer").find("canvas").should("exist")
   })
+
+  it("should create and pay an invoice", () => {
+    cy.visit(cashRegisterUrl)
+    cy.get("button[data-testid=digit-1-btn]").click()
+    cy.get("button[data-testid=digit-0-btn]").click()
+    cy.get("button[data-testid=pay-btn]").click()
+
+    cy.get("[data-testid=copy-btn]").should("exist")
+    cy.get("[data-testid=share-lbl]").should("exist")
+    cy.get("[data-testid=qrcode-container]").should("exist")
+    cy.get("[data-testid=qrcode-container]").as("qrcodeContainer")
+    cy.get("@qrcodeContainer").find("canvas").should("exist")
+    cy.get("[data-testid=copy-btn]").should("exist").click()
+
+    cy.window()
+      .then((win) => {
+        win.focus()
+        return win.navigator.clipboard.readText()
+      })
+      .then((text) => {
+        const paymentRequest = text
+        cy.log("Payment Request:", paymentRequest)
+
+        cy.loginAndGetToken(testData.PHONE, testData.CODE).then((token) => {
+          const authToken = token
+          cy.log("authToken", authToken)
+
+          cy.fetchMe(authToken).then((response) => {
+            const walletId = response.body.data.me.defaultAccount.defaultWalletId
+            cy.log("Wallet ID:", walletId)
+
+            cy.sendInvoicePayment(paymentRequest, walletId, authToken).then(
+              (paymentResponse) => {
+                expect(paymentResponse.body.data.lnInvoicePaymentSend.status).to.equal(
+                  "SUCCESS",
+                )
+              },
+            )
```

Comment:
aren't we missing a key point here on whether the QR code has been converted to a green checkmark after payment completion?